README.md

```diff
@@ -191,36 +191,33 @@ configuration to help you understand where everything plugs in.
         system = "x86_64-linux";
         modules = [
           home-manager.nixosModules.home-manager
-            {
-              home-manager.useGlobalPkgs = true;
-              home-manager.useUserPackages = true;
-              home-manager.users.my-home = (import ./my-home.nix) {
-                cornelis = cornelis.packages."x86_64-linux".cornelis;
-                cornelis-vim = cornelis.packages."x86_64-linux".cornelis-vim;
-              };
-            }
+          {
+            nixpkgs.overlays = [cornelis.overlays.cornelis];
+            home-manager.useGlobalPkgs = true;
+            home-manager.useUserPackages = true;
+            home-manager.users.my-home = import ./my-home.nix;
+          }
         ];
       };
     };
   };
 }
 
 # my-home.nix
-{cornelis, cornelis-vim}: {pkgs, ...}:
+{pkgs, ...}:
 {
-  home = {
-    packages = [cornelis pkgs.agda];
-  };
-  programs = {
-    neovim = {
-      enable = true;
-      extraConfig = builtins.readFile ./init.vim;
-      plugins = with pkgs.vimPlugins; [
-        cornelis-vim
-        vim-textobj-user
-        nvim-hs-vim
-      ];
-    };
+  home.packages = [ (pkgs.agda.withPackages (ps: [ ps.standard-library ])) ];
```

Comment:
Is the stdlib really required?

Reply:
Actually, now that you mention it, it might not be. I got an error when trying to use the plugin when I didn't include it. But that might just be because the file I was testing with required it.

Let me check.

Reply:
Yup, you were totally right. Silly me. Force pushed.